src/compose/images.ts

```diff
@@ -219,15 +219,18 @@ export class Images extends (EventEmitter as new () => ImageEventEmitter) {
 	private async withImagesFromDockerAndDB<T>(
 		cb: (dockerImages: NormalisedDockerImage[], composeImages: Image[]) => T,
 	) {
-		const images = await this.docker.listImages({ digests: true });
-		const newImages = await Bluebird.map(images, async image => {
-			const newImage = _.clone(image) as NormalisedDockerImage;
-			newImage.NormalisedRepoTags = await this.getNormalisedTags(image);
-			return newImage;
-		});
-
-		const dbImages = await this.db.models('images').select();
-		return cb(newImages, dbImages);
+		return Bluebird.join(
```

Comment:
If you want this to be async/await and parallel that can be done via
```ts
const [images, newImages] = await Promise.all([
	this.docker.listImages({ digests: true }),
	Bluebird.map(images, async image => {
		const newImage = _.clone(image) as NormalisedDockerImage;
		newImage.NormalisedRepoTags = await this.getNormalisedTags(image);
		return newImage;
	})
])
```
or the equivalent with assigning to vars and then awaiting later